hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2033,7 +2024,11 @@
       name: chip_sw_csrng_lc_hw_debug_en
       desc: '''Verify the effect of LC HW debug enable on CSRNG.
 
-            TODO: This is pending SCA security review and might be removed.
+            lc_hw_debug_en is used to diversify the csrng seed.
+            - Instantiate CSRNG with fixed seed and fetch entropy while in TEST state (hw_debug_en = 1).
```

Comment:
Trying to figure out what "fixed seed" means in this context. To text this function, a seed must be fetched from ES, but the value of the seed depends on the mode of ES and what RNG is delivery for output. Probably just need to state that a csrng instantiate with flag0 = 0 will be done.

Reply:
ahhhhhhhhhhhh... that is a critical point i missed.
Okay you're right, this does not work.